### PR TITLE
refactor: rename TripUpdate -> TripDescriptor

### PR DESCRIPTION
**Pull request by paulswartz**
In GTFS-RT, TripUpdate is the combination of a TripDescriptor and the
relevant StopTimeUpdates. This renames the Concentrate struct to more closely
match that naming.

### CODE REVIEW

**Comment by paulswartz** [comment]
@lemald `mbta/trc` auto-assigned this, but no rush.

File: lib/concentrate/encoder/gtfs_realtime_helpers.ex [hunk 1]
```diff
@@ -160,36 +160,36 @@ defmodule Concentrate.Encoder.GTFSRealtimeHelpers do
     Map.update(map, trip_id, {nil, [], [stu]}, &add_stop_time_update(&1, stu))
   end
 
-  defp add_trip_update({_tu, vps, stus}, tu) do
-    {tu, vps, stus}
+  defp add_trip_update({_td, vps, stus}, td) do
```
Comment (lemald) [inline lib/concentrate/encoder/gtfs_realtime_helpers.ex:163]: Should this become `add_trip_descriptor`?

File: lib/concentrate/encoder/gtfs_realtime_helpers.ex [hunk 2]
```diff
@@ -160,36 +160,36 @@ defmodule Concentrate.Encoder.GTFSRealtimeHelpers do
...
 
-  defp add_stop_time_update({tu, vps, stus}, stu) do
-    {tu, vps, [stu | stus]}
+  defp add_stop_time_update({td, vps, stus}, stu) do
+    {td, vps, [stu | stus]}
   end
 
   defp build_trip_update_entity(
-         {%TripUpdate{} = update, vps, stus},
+         {%TripDescriptor{} = update, vps, stus},
```
Comment (lemald) [inline lib/concentrate/encoder/gtfs_realtime_helpers.ex:176]: For clarity, maybe rename `update` to `descriptor`.

File: lib/concentrate/encoder/vehicle_positions_enhanced.ex [hunk 3]
```diff
@@ -18,7 +18,7 @@ defmodule Concentrate.Encoder.VehiclePositionsEnhanced do
     Jason.encode!(message)
   end
 
-  def build_entity({%TripUpdate{} = update, vps, _stus}) do
+  def build_entity({%TripDescriptor{} = update, vps, _stus}) do
```
Comment (lemald) [inline lib/concentrate/encoder/vehicle_positions_enhanced.ex:21]: `update` -> `descriptor`

File: lib/concentrate/encoder/gtfs_realtime_helpers.ex (hunk 1 above)
Comment (paulswartz) [reply lib/concentrate/encoder/gtfs_realtime_helpers.ex:163]: I thought this was adding the TripUpdate to the GTFS-RT file, but on second look I believe you're correct. It's building the group and adding the `TripDescriptor` to it.

File: lib/concentrate/group_filter/trip_update_timestamp.ex [hunk 4]
```diff
@@ -1,15 +1,15 @@
-defmodule Concentrate.GroupFilter.TripUpdateTimestamp do
+defmodule Concentrate.GroupFilter.TripDescriptorTimestamp do
```
Comment (lemald) [inline lib/concentrate/group_filter/trip_update_timestamp.ex:1]: Should also `git mv` the file.

Comment (paulswartz) [reply lib/concentrate/group_filter/trip_update_timestamp.ex]: Thanks, there were a few others as well.

**Review by lemald — APPROVED** [review]
👍

**Pull request merged**